### DISCUSSION TITLE
Duda sobre tiempo y uso de memoria

### DISCUSSION THREAD

**Discussion by gabo1243** [IIC2133-PUC/2024-1#107] [Tarea 1] [answered]
Hola, estaba realizando el test 6-all-crazy-wikipedia y me pasa que al darle 
```
time COMANDO
``` 
me dan los siguientes valores:
real    0m11.632s
user    0m7.857s
sys     0m0.140s

Según el enunicado debe durar menos de 10 segundos, entonces no sé si no cumplo con tal criterio ya que me da un tiempo menor a 10 y otro ligeramente mayor.

Por otra parte al hacer 
```
valgrind --tool=massif --massif-out-file=massif.out COMANDO
```
 me imprime valores de "total" en B (bytes supongo), si el parámetro "total" no llega al GB de RAM todo está bien o no? O es otro parámetro el que tengo que sacar? (sobre uso de memoria)

**Comment by benjavicente** [comment, marked as answer]
Cumples el criterio, aunque ojo que el test 6 es para el bonus (como se mencionó en el anuncio de esos tests).

Al visualizar el archivo de massif, te debería salir el peak de memoria (como en un gráfico de memoria x tiempo) y el total de memoria alocada. Lo que importa es que el peak no pase los 1Gb (siempre el peak es menor que la cantidad total alocada).
